pkg/verify/store/memory/memory.go

```diff
@@ -0,0 +1,36 @@
+// Package memory implements an in-memory signature store.  This is
```

Comment:
This feels more like another Store implementation - fileStore, httpStore, memoryStore. It could then be used to test parallel and serial too.

Reply:
It is.  But this one is already its own sub-package, so `Store` is sufficiently scoped.  I'll move HTTP/file stores into sub-packages in follow-up work.

Reply:
Yeah, I know it is a `Store` but `fileStore` and `httpStore` seem different than parallel and serial. The former and `memoryStore` being `Store`s and the later being "store managers" whereby the managers don't need a callback as they just invoke the underlying `Store`'s callback. Even though I know we currently and will probably continue to have only one callback implementation. Not advocating a major redo here, or even any changes, just explaining my line of thought.